Revit_Core_Engine/Query/ElementType.cs

```diff
@@ -125,6 +127,18 @@ public static RoofType ElementType(this BH.oM.Physical.Elements.Roof roof, Docum
 
         /***************************************************/
 
+        [Description("Returns the Revit LightingFixture type to be used when converting a given BHoM Luminaire to Revit.")]
+        [Input("luminaire", "BHoM Luminaire to find a correspondent Revit element type for.")]
+        [Input("document", "Revit document to parse in search for the element type.")]
+        [Input("settings", "Revit adapter settings to be used while performing the query.")]
+        [Output("lightingFixtureType", "Revit LightingFixture type to be used when converting the input BHoM object to Revit.")]
+        public static FamilySymbol ElementType(this BH.oM.Lighting.Elements.Luminaire luminaire, Document document, RevitSettings settings = null)
+        {
+            return (FamilySymbol)luminaire.ElementTypeInclProperty(luminaire.LuminaireType, document, luminaire.LuminaireType.BuiltInCategories(), settings);
```

Comment:
As in the previous comment, explicit casting can be dangerous here.

```suggestion
            return luminaire.ElementTypeInclProperty(luminaire.LuminaireType, document, luminaire.LuminaireType.BuiltInCategories(), settings) as FamilySymbol;
```

Reply:
fixed